app/server/appsmith-server/src/main/resources/application.properties

```
@@ -7,6 +7,7 @@ server.max-http-request-header-size=16KB
 spring.lifecycle.timeout-per-shutdown-phase=20s
 
 spring.profiles.active=${ACTIVE_PROFILE:production}
+spring.data.mongodb.uri=${APPSMITH_DB_URL:}
```

Let's please not have a default value here. If this is empty, server startup will fail, so might as well fail earlier here itself. Just like how it was in `application-production.properties`.

```suggestion
spring.data.mongodb.uri=${APPSMITH_DB_URL}
```